labgridhelper/linux.py

```diff
@@ -16,7 +16,7 @@ def get_systemd_version(command):
     out = command.run_check("systemctl --version")
     out = out[0]
 
-    parsed = re.search(r'^systemd\s+(?P<version>\d+)\s+', out)
+    parsed = re.search(r'^systemd\s+(?P<version>\d+)(\s+|$)', out)
```

Comment:
Can't we simply remove the `\s+` at the end? this way both strings can match without requiring explicit whitespace and ignoring anything after the version.

Reply:
My idea was that if they put something after the version (i.e. `systemd 234.4`) it will fail on predictable place.

If you think that it is not needed I have no problem simplifying it.

Reply:
Yes please do, the subsequent call to int will be able to handle stable release versions:
```
ython 3.8.5 (default, Jul 20 2020, 13:26:22)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int(234.5)
234
```

Thanks.